data-integration/tests/test.bal

```diff
@@ -0,0 +1,146 @@
+import data_integration.db;
+import ballerina/http;
+import ballerina/lang.runtime;
+import ballerina/os;
+import ballerina/test;
+
+final http:Client cl = check new (string `http://localhost:${port}/taskmanager`, retryConfig = {maxWaitInterval: 12});
+
+@test:AfterSuite {}
+function afterSuite() returns error? {
+    _ = check stopDBService();
```

Comment:
Again, doesn't have to be part of the test, right?

Reply:
When we're running the test cases we are starting the db service and once the tests are completed, the db should also be stopped otherwise the tests might fail the next time we run them.